import_ldraw.py

```diff
@@ -928,6 +946,41 @@ def getColorValue(line, value):
     if value in line:
         n = line.index(value)
         return line[n + 1]
+        
+# Clean-up the design by linking identical parts (mesh/colour)        
+def linkedParts():
+    colours = []
+    for colour in bpy.data.materials:
+        colours.append(colour.name)
+    
+    parts = []
+    for part in bpy.data.objects:
+        if part.type == 'MESH' and part.name.split('.')[0] not in parts:
+            parts.append(part.name.split('.')[0])
+
+    for part in parts:
+        for colour in colours:
+            replaceParts(part, colour)
+
+# Replace identical meshes of part/colour-combination with a linked version   
+def replaceParts(Part, Colour):
+    scene = bpy.context.scene
+    mat = bpy.data.materials[Colour]
+    mesh = ''
```

Comment:
Why an empty string? Is this to denote "something might be here but we're not sure?" In that case, it would better be initialized as `None`, change the checks to `if x is None` / `if x is not None` and line 978 to read `ob.data = ""`.